xds/src/main/java/io/grpc/xds/internal/rbac/engine/GrpcAuthorizationEngine.java

```diff
@@ -316,9 +335,39 @@ private String getPath() {
 
     @Nullable
     private String getHeader(String headerName) {
-      if (headerName.endsWith(Metadata.BINARY_HEADER_SUFFIX)) {
+      if ("TE".equals(headerName)) {
         return null;
       }
+      if ("Host".equals(headerName)) {
+        headerName = "authority";
```

Comment:
I thought this should be `:authority`, but we do not allow `:` to be part of the key:
```
    java.lang.IllegalArgumentException: Invalid character ':' in key name ':authority'
        at com.google.common.base.Preconditions.checkArgument(Preconditions.java:273)
        at io.grpc.Metadata$Key.validateName(Metadata.java:742)
```
what do i miss?

Reply:
The authority is not part of the metadata. It is available at `ServerCall.getAuthority()`